src/include/simeng/pipeline/FetchUnit.hh

```diff
@@ -37,6 +64,9 @@ class FetchUnit {
    * branch. */
   uint64_t getBranchStalls() const;
 
+  /** Clear the loop buffer. */
+  void flush();
+
```

Comment:
Given this is in `FetchUnit.hh`, perhaps this should be renames to `flushLoopBuffer()` to avoid any possible confusion